frontend/pages/hosts/ManageHostsPage/components/HostsFilterBlock/HostsFilterBlock.tsx

```diff
@@ -543,6 +556,38 @@ const HostsFilterBlock = ({
     );
   };
 
+  const renderScriptBatchExecutionBlock = () => {
+    const OPTIONS = [
+      { value: "ran", label: "Ran" },
+      { value: "errored", label: "Error" },
+      { value: "pending", label: "Pending" },
+    ];
+    return (
+      <>
+        <Dropdown
+          value={scriptBatchExecutionStatus}
+          className={`${baseClass}__script-batch-status-dropdown`}
+          options={OPTIONS}
+          searchable={false}
+          onChange={onChangeScriptBatchStatusFilter}
+          iconName="filter-alt"
+        />
+        {scriptBatchScriptName && (
+          <FilterPill
+            label={scriptBatchScriptName}
+            onClear={() =>
+              handleClearFilter([
+                HOSTS_QUERY_PARAMS.SCRIPT_BATCH_EXECUTION_ID,
+                HOSTS_QUERY_PARAMS.SCRIPT_BATCH_EXECUTION_STATUS,
+              ])
+            }
+            tooltipDescription={scriptBatchRanAt}
```

Comment:
This tooltip functionality is currently broken across all instances per [this unreleased bug](https://github.com/fleetdm/fleet/issues/29613) - that resolution should fix this

Reply:
Can we get this in "time ago" format?  (looks like we use a `dateAgo` utility)